src/nodejs/neon-js/modules/get_last_transactions_by_address.js

```diff
@@ -0,0 +1,71 @@
+// neoscan get_last_transactions_by_address cli wrapper
+
+require('module-alias/register')
+
+const program = require('commander');
+const _       = require('underscore')
+
+const dbg     = require('nodejs_util/debug')
+const neoscan = require('nodejs_neoscan/neoscan')
+const email   = require('nodejs_alert/email')
+
+const get_last_transactions_by_address = require('nodejs_neoscan/modules/get_last_transactions_by_address')
+
+var cfg       = require('nodejs_config/config.js')
+var config    = cfg.load('nodejs_config/nodejs.config.json')
+
+let defly = false
+let address
+
+function print(msg) {
+  console.log(msg)
```

Comment:
same thing here - If you'd like to use the function per se, I'd suggest moving it to the debug section maybe? 
Duplicate code is never good :)